sdk/resourcemanager/Azure.ResourceManager.Core/src/SubscriptionOperations.cs

```diff
@@ -94,6 +94,15 @@ public virtual LocationContainer GetLocations()
             return new LocationContainer(this);
         }
 
+        /// <summary>
+        /// Gets the provider container under this subscription.
+        /// </summary>
+        /// <returns> The provider container. </returns>
+        public ProviderContainer GetProviders()
```

Comment:
Didn't we walk about adding this at both the TenantOperations and SubscriptionOperations level?